.gitignore

```diff
@@ -15,4 +16,4 @@ Thumbs.db
 .idea
 .vscode
 *.sublime-project
-*.sublime-workspace
\ No newline at end of file
+*.sublime-workspace
```

Comment:
Please exclude this from commit, we use `webpack-defaults` for all repos, need do PR there https://github.com/webpack-contrib/webpack-defaults